addons/web/static/tests/public/interaction.test.js

```diff
@@ -124,6 +124,35 @@ describe("adding listeners", () => {
         await click("span");
         expect.verifySteps(["setup", "willStart", "start", "click"]);
     });
+
+    test("listener is added on iframe single element", async () => {
+        class Test extends Interaction {
+            static selector = "iframe";
+        }
```

Comment:
you can simply do all that stuff in the start method: 
```js
class Test extends Interaction {
static selector = "iframe";
start() {
  const spanEl = this.el.contentDocument.createElement("span");
  spanEl.textContent = "abc";
  this.el.contentDocument.body.appendChild(spanEl);
  this.addListener(spanEl, "click", () => expect.step("click"));
  spanEl.click();
}
```
i think that it makes the test easier to understand